avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -660,15 +671,27 @@ def updatePositionC(cfg, particles, dem, force, DT):
 
     TotkinEneNew = TotkinEneNew + 0.5 * m * uMag * uMag
     TotpotEneNew = TotpotEneNew + mNew * gravAcc * zNew
+    TotForceSPHNew = TotForceSPHNew + mNew * norm(forceSPHX[j], forceSPHY[j], forceSPHZ[j])
+
+    if idrel == 0:
```

Comment:
Please add a quick comment what idrel represents or what this if if needed for

Reply:
done